Dockerfile

```diff
@@ -43,4 +43,5 @@ WORKDIR /app/
 COPY --from=development /build .
 RUN apk update && apk add iproute2 curl
 
-ENTRYPOINT ["./bittwister", "start", "-d", "eth0", "-p", "50"]
+ENV SERVE_ADDR="localhost:9001"
+ENTRYPOINT ["sh", "-c", "./bittwister serve --serve-addr $SERVE_ADDR"]
```

Comment:
Thanks will change it to `CMD`

the `sh -c` is there because of that ENV var. docker does not read it when we pass it in the array format. so the only working way that I found was this way:

```bash
ENV SERVE_ADDR="localhost:9001"
CMD ["sh", "-c", "./bittwister serve --serve-addr ${SERVE_ADDR}"]
```

I'd use ENV instead of ARG as ARG is used in build time and in a large scale test, we build it once and then create multiple containers out of the image and so we can pass the ENV when creating the container.